lib/controllers/multi-file-patch-controller.js

```diff
@@ -183,30 +188,47 @@ export default class FilePatchController extends React.Component {
   }
 
   async discardRows(rowSet, nextSelectionMode, {eventSource} = {}) {
+    // (kuychaco) For now we only support discarding rows for MultiFilePatches that contain a single file patch
```

Comment:
super, duper nit: is there a reason this comment is directed at Katrina?

Reply:
This is true ☝️. Maybe there's a more clear format for this?